cli/command/image/build/context.go

```diff
@@ -219,6 +224,8 @@ func getDockerfileRelPath(givenContextDir, givenDockerfile string) (absContextDi
 				absDockerfile = altPath
 			}
 		}
+	} else if absDockerfile == "-" {
```

Comment:
This is called before the name replacement. So the return value is replaced anyway, but would be nice not to return anything broken from the func. So it returns the default as it can't determine the relative Dockerfile path.